### PR TITLE
doc: add framework and docs

### PR DESCRIPTION
**Pull request by greg-fer**
Added documentation framework and documentation for the app.

### CODE REVIEW

**Review by KievDevel — APPROVED** [review]

File: doc/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      nRF Connect Direct Test Mode cannot be used as a testing or certification tool. This is because it only provides a graphical wrapper around DTM commands for the Upper Tester. For verification using both the Upper Tester and the Lower Tester, you need an external tester equipment. For example, in case of the [Direct Test Mode Bluetooth sample](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/direct_test_mode/README.html), this can be the Anritsu MT8852 or similar, in which case you obtain the following configuration:
+
+      ![Testing setup for the Direct Test Mode Bluetooth sample](./screenshots/bt_dtm_dut.svg "Testing setup for the Direct Test Mode Bluetooth sample")
+
+nRF Connect Direct Test Mode is installed and updated using [nRF Connect for Desktop](https://docs.nordicsemi.com/bundle/ug_nrf_connect_desktop/page/struct/nrftools_nrfconnect.html) (v4.4.1 or later).
+
+## Supported devices
+
+- nRF52840 DK
+- nRF52833 DK (nRF52 DK)
```
Comment (kylebonnici) [inline doc/docs/index.md:25]: We also support 3rd Part devices as long as they have the DTM standard protocol programmed on them already and expose this via COM port

File: doc/docs/using_dtm.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          The [Direct Test Mode Bluetooth sample](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/direct_test_mode/README.html) supports two Device Under Test communication protocols: 2-wire UART and experimental HCI UART interface. nRF Connect Direct Test Mode only supports the 2-wire UART.
+
+* **Device 2**: One device programmed with the firmware conformant to the Bluetooth specification.</br>
+  This can also be an off-the-shelf product certified for Bluetooth 5.3.
+
+## Transmitting
+
+To execute transmitting, complete the following steps:
+
+1. Connect the **Device 1** to the computer using a USB cable. The device is assigned a `COM` port (Windows) or `ttyACM` device (Linux), which is visible in the Device Manager.
```
Comment (kylebonnici) [inline doc/docs/using_dtm.md:23]: What about MAC os? 'tty.usbmodem' ... that is what I see nt sure if this is a given tough

File: doc/docs/using_dtm.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+1. Program or start (or both) the receiver on the **Device 2**.</br>
+1. Start the Direct Test Mode app in nRF Connect for Desktop.
+1. In the Direct Test Mode app:
+
+    1. Select the supported device programmed with the firmware compatible with Direct Test Mode.
```
Comment (kylebonnici) [inline doc/docs/using_dtm.md:28]: The app will offer to program FW for the PCA10040 and PCA 10056

File: doc/docs/using_dtm.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    1. Configure the [transmitter settings](overview.md#transmitter-tab) to the ones matching the **Device 2**.
+    1. Start the test.
+    1. On the application chart, observe that the number of TX packets is increasing.
+    1. Stop the test.
+
+## Receiving
+
+To execute receiving, complete the following steps:
+
+1. Connect the **Device 1** to the computer using a USB cable. The device is assigned a `COM` port (Windows) or `ttyACM` device (Linux), which is visible in the Device Manager.
```
Comment (kylebonnici) [inline doc/docs/using_dtm.md:39]: Same as above

File: doc/docs/using_dtm.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+1. Program or start (or both) the transmitter on the **Device 2**.</br>
+1. Start the Direct Test Mode app in nRF Connect for Desktop.
+1. In the Direct Test Mode app:
+
+    1. Select the supported device programmed with the firmware compatible with Direct Test Mode.
```
Comment (kylebonnici) [inline doc/docs/using_dtm.md:44]: Same as a above

File: doc/docs/using_dtm.md (hunk 3 above)
Comment (greg-fer) [reply doc/docs/using_dtm.md:28]: This I mentioned in the tip at the top of the page.

File: doc/docs/using_dtm.md (hunk 5 above)
Comment (greg-fer) [reply doc/docs/using_dtm.md:44]: This I mentioned in the tip at the top of the page.

**Review by kylebonnici — APPROVED** [review]

**Pull request merged**